frontend/src/Frontend/UI/Dialogs/Send.hs

```diff
@@ -335,48 +335,62 @@ sendConfig model initData = Workflow $ do
           displayImmediateFeedback (updated decoded) cannotBeReceiverMsg
             $ either (const False) (\ka -> _txBuilder_accountName ka == fromName && _txBuilder_chainId ka == fromChain)
 
-          (_, amount, _) <- mkLabeledInput True "Amount" uiGasPriceInputField
-            (def & inputElementConfig_initialValue .~ (maybe "" tshow mInitAmount))
-            -- We're only interested in the decimal of the gas price
-            <&> over (_2 . mapped . mapped) (\(GasPrice (ParsedDecimal d)) -> d)
+          let
+            balance = _account_status fromAcc ^? _AccountStatus_Exists . accountDetails_balance . to unAccountBalance
+
+            gasInputWithMaxButton cfg = mdo
+              let projectAmount = over (_2 . mapped . mapped) (\(GasPrice (ParsedDecimal d)) -> d)
```

Comment:
Could you not get the Wrapped instance to work either? It was doing weird things for me.

Reply:
Ah didn't really try, just moved this in from outside

Reply:
Ahh, cool.

Reply:
`_Wrapped` was acting up but `_Wrapped'` worked